Description: 64-bit time_t compatibility for v4l2 module
 The v4l2 build unsets _FILE_OFFSET_BITS, which is not allowed when setting
 _TIME_BITS=64.  Having verified that nothing in this module is sensitive to
 64-bit time_t (none of the functions it intercepts handle time), we also
 unset _TIME_BITS to allow this to build as before.
Author: Steve Langasek <steve.langasek@canonical.com>
Forwarded: no
Last-Update: 2024-03-13
Bug-Debian: https://bugs.debian.org/1066836

--- a/src/v4l2/meson.build
+++ b/src/v4l2/meson.build
@@ -23,6 +23,7 @@
     # file operations, disable transparent large file support.
     '-U_FILE_OFFSET_BITS',
     '-D_FILE_OFFSET_BITS=32',
+    '-U_TIME_BITS',
     '-D_LARGEFILE64_SOURCE',
     '-fvisibility=hidden',
 ]
